src/main/java/uk/gov/companieshouse/company/profile/api/InsolvencyApiService.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.companieshouse.company.profile.api;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.chskafka.ChangedResource;
+import uk.gov.companieshouse.api.chskafka.ChangedResourceEvent;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.chskafka.request.PrivateChangedResourcePost;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.logging.Logger;
+
+@Service
+public class InsolvencyApiService {
+
+    private static final String CHANGED_RESOURCE_URI = "/resource-changed";
+    private final Logger logger;
+    private final String chsKafkaUrl;
+    private final ApiClientService apiClientService;
+
+    /**
+     * Invoke Insolvency API.
+     */
+    public InsolvencyApiService(@Value("chs.kafka.api.endpoint")String chsKafkaUrl,
+                                ApiClientService apiClientService, Logger logger) {
+        this.chsKafkaUrl = chsKafkaUrl;
+        this.apiClientService = apiClientService;
+        this.logger = logger;
+    }
+
+    /**
+     * Call chs-kafka api.
+     * @param companyNumber company insolvency number
+     * @return response returned from chs-kafka api
+     */
+    public ApiResponse<?> invokeChsKafkaApi(String companyNumber) {
```

Comment:
Not sure why '?' here? we only be interested if we get 2xx response back or not in this call

Reply:
Changes done